admin/assets/javascripts/util-checkbox.js

```diff
@@ -279,12 +281,23 @@ var checkboxUtil = {
     */
   tableRowVisibility: function (param, paramIds) {
     var sel = document.getElementsByClassName('govuk-template').length === 1 ? '.govuk-spacious > tbody > tr' : '.spacious > tbody > tr'
+    const checkBoxEls = document.querySelectorAll(sel.concat(' > td > .multiple-choice-mtc'))
     if (paramIds.length < 1 || paramIds[0].length < 1) {
       $(sel).removeClass('filter-hidden-group')
+      for (let i = 0; i < checkBoxEls.length; i++) {
+        checkBoxEls[i].classList.remove('filter-hidden-group')
+      }
     } else {
       $(sel).addClass('filter-hidden-group')
+      for (let i = 0; i < checkBoxEls.length; i++) {
+        checkBoxEls[i].classList.add('filter-hidden-group')
```

Comment:
If this class `filter-hidden-group` is only applied so that javascript can use it, then it should be prefixed with `js-`

Reply:
added